lametro/templatetags/lametro_extras.py

```diff
@@ -144,4 +144,16 @@ def parse_agenda_item(text):
         label, number = text.split(',')
         return number
     else: 
-        return ''
\ No newline at end of file
+        return ''
+
+
+@register.filter
+def updates_made(event_id):
```

Comment:
Please add a docstring to explain why we want to know when an event was updated after a document associated with that event?